### ISSUE TITLE
Minor correction in the readme-developer.txt

### ISSUE THREAD

**Issue by lyubod** [endless-sky/endless-sky#4834]
The scons build of Endless Sky 0.9.2 fails on Debian Stretch based distributions, for lack of a package libjpeg-turbo8-dev.

The readme-developer.txt lists a build dependency on, among other files:

libjpeg-turbo8-dev

which does not exist on distributions based on Debian Stretch, or newer. The package is called libjpeg62-turbo-dev up to Debian Buster, and the name is almost certain to change in the future when it gets updated to keep track of new libjpeg and Debian releases.

I believe the more error free and appropriate method to list it, at least for Debian (in case it is called libjpeg-turbo8-dev in other distributions) is to call the dummy package libjpeg-dev, which installs whichever version is current for your flavor of Debian.

Here is a proposed diff for version 0.9.2, also attached as file below:

--- readme-developer.txt	2016-06-24 15:15:47.000000000 +0300
+++ /home/lubod/Downloads/endless-sky-0.9.2-OpenGL2.1/readme-developer.txt	2020-02-20 14:51:04.327851880 +0200
@@ -14,6 +14,9 @@
    libgl1-mesa-dev \
    libglew-dev \
    libopenal-dev
+   
+   libjpeg-turbo8-dev replaced by libjpeg62-turbo-dev in Debian 9 "Stretch" or newer,
+   and is provided by the dummy package libjpeg-dev
 
 You can then just navigate to the source code folder in a terminal and type:

**Comment by lyubod** [comment]
I see this has been fixed in the master branch:

https://github.com/endless-sky/endless-sky/commit/04b99dc89a40b584151202abab2379d52c7bf0b9#diff-d195a0c322055d3cc6c95452c9102bd2

So it only affects older versions of Debian and older builds of Endless Sky.

**Comment by tehhowch** [comment]
Ref #4630 
Packaging the game for each of the many flavors of Linux is not something that is done by us directly. You are welcome to submit a newer package (0.9.10) for Stretch; we will be releasing an unstable version in the next few days and 0.9.12 a month-ish later.

**Comment by lyubod** [comment]
Thank you for the reply.

Since this only affects older Debian releases, and older builds of
Endless Sky, I had no illusions that it would not be marked "Won't
Fix". Ubuntu and company, everyone who insists everything must be 64
bit and modern (whatever that means to them), do this, almost always.

To see that someone actually cares about older systems is a pleasant surprise.

I am more a system administrator and end user, with some experience in
compiling other people's code, but quite new to packaging, so I can't
(yet) contribute a package. I compiled it OK. But I have no .deb, yet.
I saw that one of the "suggested" packages when building Endless Sky
is "debhelper", but I'm can't make it work to my satisfaction, yet.
Been using Ubuntu since the "unalphabetical" Warty Warthog circa 2004,
and Debian circa 1998 (on a 68K Mac). The administration and use is
moderately easy. The packaging seems a magic ritual, with many complex
steps, which fail (or work) unpredictably. :-)

Endless Sky is my example to silence those who say "there are no good
Linux games". Of course it is sad that almost all Linux games are
ported to windows sooner or later, but the opposite (except for
companies like ID or Loki), rarely happens.
